packages/common/src/authentication/api/api.ts

```diff
@@ -29,8 +29,7 @@ const authTokenGuard = (
 
   if (authTokenQuery?.authToken?.__typename === 'AuthTokenError') {
     switch (authTokenQuery.authToken.error.__typename) {
-      case 'DatabaseError':
-      case 'InternalError':
+      case 'InvalidCredentials':
```

Comment:
This would return the message from the server, which is not the intention here. We need to have it translatable, which won't be provided by the server.

The original was catching systemic errors and showing those, and with other Auth errors ( e.g. InvalidCredentials ) showing the message from `app.json` : `error.login`

Given that we don't have access to `useTranslation` here, I'd recommend returning 
```
{
          token: '',
          error: { message: '' },
}
```

To return the original, translated string. Otherwise it shows this regardless of language:

<img width="252" alt="Screen Shot 2022-04-05 at 10 36 54 AM" src="https://user-images.githubusercontent.com/9192912/161643128-a147c80d-a38c-40a9-b9fd-4f0902e4e2f5.png">

otherwise, a specific token to indicate there was an auth error. Or always show the invalid user - though it is helpful to also show if the server is down, which is what the last part will do.

Reply:
Ah yes, good point. Thanks df398907